index.html

```diff
@@ -203,6 +203,41 @@ <h1 class="introduction">I’m Mariane Glad to see you!</h1>
       <li class="topic item">Skills</li>
     </ul>
   </div>
+
+  <footer class="footer">
+    <div class="contact-me">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I’d love to help with it
+      </p>
+    </div>
+
+    <div class="contact-forms">
+      <form class="message-form" action="https://formspree.io/f/form_id" method="post">
```

Comment:
- [ ] Your form is designed well 👏. However, it is not being submitted to formspree platform. This is because maybe you have not created an account there so that you get the ID that you will be using in your forms. If you have created an account there, please add the ID they have given you in that `action` attribute link where there is a word `form_id`. By doing this, your form data will be sent successfully to the Formspree platform 😃 😎.